security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -827,6 +835,33 @@ public boolean accessTokenIpCheck() {
         return accessTokenIpCheck;
     }
 
+    /**
+     * Whether PKCE is enabled.
```

Comment:
Whenver we use an abbreviation, there should be at least one place where it is explained - why, what?